### PR TITLE
Added some Persian channels

### PR DESCRIPTION
**Pull request by xmha97**

### CODE REVIEW

File: channels/ir.m3u [hunk 1]
```diff
@@ -1,71 +1,163 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/qQBqGGf.png" group-title="General",IRIB TV5
+https://yjc.arvanlive.com/hls/tv5/tv5.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/vJN95kP.png" group-title="Entertainment",IRIB NASIM
+https://yjc.arvanlive.com/hls/nasim/nasim.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/D3tLNue.png" group-title="News",IRINN
+https://yjc.arvanlive.com/hls/irinn/irinn.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/fFnEHc7.png" group-title="Sport",IRIB Varzesh
+https://yjc.arvanlive.com/hls/varzesh/varzesh.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/RcgJiJD.png" group-title="Local",IRIB Sahand
+https://yjc.arvanlive.com/hls/sahand/sahand.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: None of these channels work for me

File: channels/ir.m3u [hunk 2]
```diff
@@ -1,71 +1,163 @@
...
+https://stream.alkafeel.net/live/alkafeel/rAa5PGot4/manifest.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala Live 5
+https://stream.alkafeel.net/live/alkafeel/rAa5PGot5/manifest.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/6/63/Karbala_tv.jpg" group-title="Religious",Karbala Live 6
+https://stream.alkafeel.net/live/alkafeel/rAa5PGot6/manifest.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/UFV7YVy.jpg" group-title="",Andisheh TV
+https://livestream.5centscdn.com/cls090117/49ead3a7c6432c5358a086d078110c16.sdp/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.parsatv.com/index_files/channels/channelone.jpg" group-title="",Channel One
+https://livestream.5centscdn.com/cls040317/0070c5b7ef083bdc8de09065c61a34d8.sdp/tracks-v1a1/mono.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: Link is broken. Tested in VLC Player and streamtest.in

File: channels/ir.m3u [hunk 3]
```diff
@@ -1,71 +1,163 @@
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://live-tv-channels.org/pt-data/uploads/logo/us-tv-azadi-4338.jpg" group-title="",TV Azadi
+https://5a3e964ed0b51.streamlock.net/w/azadah/chunklist_w1398305128.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.parsatv.com/index_files/channels/icctv.jpg" group-title="Religious",ICC TV
+https://mn-nl.mncdn.com/ezcdn_icc/icc_1/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://live-tv-channels.org/pt-data/uploads/logo/ca-parnian-tv-9509.jpg" group-title="",Parnian TV
+https://live2.parnian.tv/hls/.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://www.parsatv.com/index_files/channels/batv.png" group-title="",Bazme Asheghan
+https://593aed234297b.streamlock.net/saeed_nl/saeed_nl/chunklist_w1960247291.m3u8
```
Comment (freearhey) [inline channels/ir.m3u]: Link is broken. Tested in VLC Player and streamtest.in

**Review by freearhey — CHANGES_REQUESTED** [review]
Please replace or remove broken links from PR so that we can add them to the rest.

**Comment by amirrammari** [comment]
I removed broken links from PR, Please add them to the rest. Thanks.

**Review by freearhey — APPROVED** [review]

**Pull request merged**